This patch is part of a series of patches for festival for
Debian GNU/Linux, to use the installed speech-tools headers

Index: b/src/modules/MultiSyn/inst_tmpl/hash_s_itemlistp_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/hash_s_itemlistp_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/hash_s_itemlistp_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -51,7 +51,7 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_THash.cc"
+#include "/usr/include/speech_tools/base_class/EST_THash.cc"
 
 Instantiate_TStringHash_T(ItemList *, ItemListP)
 
Index: b/src/modules/MultiSyn/inst_tmpl/hash_itemp_tcdatap_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/hash_itemp_tcdatap_t.cc	2006-07-08 08:01:28.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/hash_itemp_tcdatap_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -50,7 +50,7 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_THash.cc"
+#include "/usr/include/speech_tools/base_class/EST_THash.cc"
 
 Instantiate_THash_T(EST_Item*, TCData*, TCDataP)
 
Index: b/src/modules/MultiSyn/inst_tmpl/list_uttp_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/list_uttp_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/list_uttp_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -51,8 +51,8 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_TList.cc"
-#include "../base_class/EST_TSortable.cc"
+#include "/usr/include/speech_tools/base_class/EST_TList.cc"
+#include "/usr/include/speech_tools/base_class/EST_TSortable.cc"
 
 Instantiate_TList_T_MIN(EST_Utterance *, EST_UtteranceP)
 
Index: b/src/modules/MultiSyn/inst_tmpl/list_itemp_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/list_itemp_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/list_itemp_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -51,8 +51,8 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_TList.cc"
-#include "../base_class/EST_TSortable.cc"
+#include "/usr/include/speech_tools/base_class/EST_TList.cc"
+#include "/usr/include/speech_tools/base_class/EST_TSortable.cc"
 
 Instantiate_TList_T(EST_Item *, EST_ItemP)
 
Index: b/src/modules/MultiSyn/inst_tmpl/list_voicemodulep_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/list_voicemodulep_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/list_voicemodulep_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -50,8 +50,8 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_TList.cc"
-#include "../base_class/EST_TSortable.cc"
+#include "/usr/include/speech_tools/base_class/EST_TList.cc"
+#include "/usr/include/speech_tools/base_class/EST_TSortable.cc"
 
 Instantiate_TList_T_MIN(DiphoneVoiceModule *, DiphoneVoiceModuleP)
 
Index: b/src/modules/MultiSyn/inst_tmpl/list_strlist_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/list_strlist_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/list_strlist_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -46,6 +46,6 @@
 Declare_TList_T(EST_TList<EST_String>,STR_LIST)
 
 #if defined(INSTANTIATE_TEMPLATES)
-#include "../base_class/EST_TList.cc"
+#include "/usr/include/speech_tools/base_class/EST_TList.cc"
   Instantiate_TList_T(EST_StrList,STR_LIST);
 #endif
Index: b/src/modules/MultiSyn/inst_tmpl/vector_jccp_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/vector_jccp_t.cc	2004-05-27 08:53:09.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/vector_jccp_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -50,8 +50,8 @@
 
 #if defined(INSTANTIATE_TEMPLATES)
 
-#include "../base_class/EST_TSimpleVector.cc"
-#include "../base_class/EST_TVector.cc"
+#include "/usr/include/speech_tools/base_class/EST_TSimpleVector.cc"
+#include "/usr/include/speech_tools/base_class/EST_TVector.cc"
 
 Instantiate_TVector_T_MIN(EST_JoinCostCache*,EST_JoinCostCacheP)
 Instantiate_TSimpleVector(EST_JoinCostCache*)
Index: b/src/modules/MultiSyn/inst_tmpl/list_scorepair_t.cc
===================================================================
--- a/src/modules/MultiSyn/inst_tmpl/list_scorepair_t.cc	2006-07-03 10:40:06.000000000 -0500
+++ b/src/modules/MultiSyn/inst_tmpl/list_scorepair_t.cc	2011-02-16 10:42:27.000000000 -0600
@@ -45,8 +45,8 @@
 Declare_TSortable(ScorePair)
 
 #if defined(INSTANTIATE_TEMPLATES)
-#include "../base_class/EST_TList.cc"
-#include "../base_class/EST_TSortable.cc"
+#include "/usr/include/speech_tools/base_class/EST_TList.cc"
+#include "/usr/include/speech_tools/base_class/EST_TSortable.cc"
 Instantiate_TList(ScorePair);
 Instantiate_TSortable(ScorePair);
 #endif
